src/test/scala/org/allenai/deep_qa/data/OmnibusDaDatasetReaderSpec.scala

```diff
@@ -21,11 +23,54 @@ class OmnibusDaDatasetReaderSpec extends FlatSpecLike with Matchers {
   val questionText2 = "Coal is a nonrenewable energy resource. People burn coal to " +
     "make electricity and heat buildings. Explain how long it would most likely take for new coal " +
     "to form."
+  val transformedQuestionText2 = "Coal is a nonrenewable energy resource. People burn coal to " +
+    "make electricity and heat buildings. How long it would most likely take for new coal to form."
```

Comment:
This is getting increasingly brittle, but you could consider a simple regex to replace the order of "would" and "it" here.  I'd want to see more examples of this before deciding on whether this was a good idea and how to do it, though.